pkg/binding/controller.go

```diff
@@ -506,14 +512,14 @@ func (c *Controller) reconcile(ctx context.Context, key util.Key) error {
 			// will add name.
 		}
 
-		c.logger.Info("handled bindingpolicy", "object", util.RefToRuntimeObj(obj))
+		logger.Info("handled bindingpolicy", "object", util.RefToRuntimeObj(obj))
 		return nil
 	} else if util.IsCRD(obj) {
 		if err := c.handleCRD(obj); err != nil {
 			return fmt.Errorf("failed to handle CRD: %w", err) // error logging after this call
 			// will add name.
 		}
-		c.logger.Info("handled CRD", "object", util.RefToRuntimeObj(obj))
+		logger.Info("handled CRD", "object", util.RefToRuntimeObj(obj))
```

Comment:
Logger message should start with a capital letter like the rest of the changed messages.

Reply:
fixed